node/resolvers/directives/checkAdminAccess.ts

```diff
@@ -18,6 +20,24 @@ export class CheckAdminAccess extends SchemaDirectiveVisitor {
         clients: { identity },
       } = context
 
+      // send metric to collect data about caller and tokens used in the request
+      const operation = field.astNode?.name?.value ?? context.request.url
```

Comment:
🟢 
Scripts are not my favorite languages.
Suggest to add` ?` In all fields navigation to avoid a a code break because of metrics.